### PR TITLE
Added more cmdline options

### LINKED ISSUE #478: Feature request: open geckodriver / chromedriver with a CLI specified URL:PORT
While resolving Issue #474 I discovered that there is no predetermined way to choose the driver or the port from the CLI. 

So, if I could have my way, I would love the command to be something like:
```
$ zap ./zap/ZAP_D-2019-04-29/zap.sh -daemon \ 
-host 127.0.0.1 -port 7777 -hud -nostdout
-driver firefox -url ${serverUrl}
```

I can verify, however, that my automated installation process for Quasar 1.0 works (at least on my Mac): https://github.com/quasarframework/quasar-testing/releases/tag/v1.0.0-beta.5

### PR DESCRIPTION
**Pull request by psiinon**
Fixes #478
New options '-hudurl <url>' and '-hudbrowser <browser>'

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: src/main/java/org/zaproxy/zap/extension/hud/launch/ExtensionHUDlaunch.java [hunk 1]
```diff
@@ -46,8 +50,15 @@
...
-    private CommandLineArgument[] arguments = new CommandLineArgument[1];
-    private static final int ARG_BROWSER_IDX = 0;
+    private CommandLineArgument[] arguments = new CommandLineArgument[3];
+    private static final int ARG_HUD_IDX = 0;
+    private static final int ARG_HUD_URL_IDX = 1;
+    private static final int ARG_HUD_BROWSER_IDX = 2;
+
+    private static final String FIREFOX = "Firefox";
+    private static final String CHROME = "Chrome";
+    private static final Set<String> SUPPOORTED_BROWSERS =
```
Comment (kingthorin) [inline src/main/java/org/zaproxy/zap/extension/hud/launch/ExtensionHUDlaunch.java]: Typo on support 😉

Comment (psiinon) [reply src/main/java/org/zaproxy/zap/extension/hud/launch/ExtensionHUDlaunch.java]: ha!
Also just remembered that I'm not updated the help :(
Will update for both of those...

Comment (psiinon) [reply src/main/java/org/zaproxy/zap/extension/hud/launch/ExtensionHUDlaunch.java]: Done both of those :)

**Comment by nothingismagick** [comment]
Awesome!!!

>

**Pull request merged**